### PR TITLE
feat(CX-3173): Add header to Collector Profile

### PR DESCRIPTION
**Pull request by MrSltun**
The type of this PR is: **Feature**

This PR solves [CX-3173]

### Description
This PR adds the header to the Collector Profile page

### Screeshots
| mWeb | Desktop | 
| -- | -- |
| image | image |

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3173]: https://artsyproduct.atlassian.net/browse/CX-3173?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/CollectorProfile/Components/CollectorProfileHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          </Button>
+        </Media>
+      </Flex>
+
+      <Spacer mt={2} />
+
+      <Box mb={2}>
+        {!!bio && <Text variant={["xs", "sm-display"]}>{bio}</Text>}
+
+        <Spacer mt={[1, 2]} />
```
Comment (olerichter00) [inline src/Apps/CollectorProfile/Components/CollectorProfileHeader.tsx:67]: **question:** Do we need this separator if there is no bio? I guess we should hide it in this case.

File: src/Apps/CollectorProfile/Components/CollectorProfileHeaderInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  MuseumIcon,
+  Text,
+} from "@artsy/palette"
+import { FC, Fragment } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Media } from "Utils/Responsive"
+import { CollectorProfileHeaderInfo_me$data } from "__generated__/CollectorProfileHeaderInfo_me.graphql"
+
+const M_ICON_SIZE = 14
+const D_ICON_SIZE = 18
```
Comment (olerichter00) [inline src/Apps/CollectorProfile/Components/CollectorProfileHeaderInfo.tsx:15]: **suggestion:** Maybe we can give this a better name. I'm not sure if I understand intuitively what `M_` and `D_` means.

**Review by olerichter00 — APPROVED** [review]
Looks great 🌟 

I've just added two comments.

File: src/Apps/CollectorProfile/Components/CollectorProfileHeader.tsx (hunk 1 above)
Comment (MrSltun) [reply src/Apps/CollectorProfile/Components/CollectorProfileHeader.tsx:67]: Good catch! I'll update it now :)

**Review by olerichter00 — APPROVED** [review]

**Review by lordkiz — APPROVED** [review]
Looks good! 🎉

**Pull request merged**